src/igraph/__init__.py

```diff
@@ -42,6 +42,7 @@
 from igraph.layout import *
 from igraph.matching import *
 from igraph.operators import *
+from igraph.sparse_matrix import *
```

Comment:
Make sure to refer to your function here explicitly if you remove it from `__all__`.

I plan to remove the wildcard imports in the long term anyway, it makes linters impossible to follow what is being imported from where and they do not print warning anymore if I mistype something while editing the file.

Reply:
done that